requirements.txt

```diff
@@ -0,0 +1,7 @@
+lxml
+bleach==1.4.2
+oauthlib==1.0.3
+Mako
+git+https://github.com/edx/XBlock.git#egg=XBlock
```

Comment:
AFAIK, there's no official policy around guaranteeing functionality between versions. Certainly, if we needed to change something that might have backwards-compatibility issues (to fix a bug or enhance functionality), we wouldn't want it to affect current installations. If we don't use tagged GitHub versions, it means we can never merge any of those types of changes back to the GitHub default branch (master). We should move to a policy of pinning to a GitHub tag or hash.